.github/actions/buildresources/action.yml

```diff
@@ -14,12 +14,13 @@ inputs:
   token:
     description: 'A GitHub Personal Access Token (for publishing)'
     required: false
+    default: '0'
 
 runs:
   using: "composite"
   steps:
     - name: Setup JupyterBook Cache
-      if: ${{inputs.jb-cache}} == true
+      if: inputs.jb-cache == 'true'
```

Comment:
This was interesting to debug and research:
https://github.com/actions/runner/issues/1483